tests/src/test/java/alluxio/server/ft/MasterFaultToleranceIntegrationTest.java

```diff
@@ -260,8 +260,9 @@ public void killStandby() throws Exception {
   public void queryStandby() throws Exception {
     List<InetSocketAddress> addresses = mMultiMasterLocalAlluxioCluster.getMasterAddresses();
     Collections.shuffle(addresses);
-    PollingMasterInquireClient inquireClient =
-        new PollingMasterInquireClient(addresses, ServerConfiguration.global());
+    PollingMasterInquireClient inquireClient = new PollingMasterInquireClient(addresses,
+        ServerConfiguration.global(),
+        FileSystemContext.create(ServerConfiguration.global()).getClientContext().getUserState());
```

Comment:
is this allowed to use `ServerUserState`?

Reply:
It should ideally use a "client" user state but for this test specifically it does not make any difference.